consumer-java/aws-kinesis-video-producer-sdk-canary-consumer/src/main/java/com/amazon/kinesis/video/canary/consumer/Terminate.java

```diff
@@ -0,0 +1,30 @@
+package com.amazon.kinesis.video.canary.consumer;
+
+import com.amazonaws.kinesisvideo.parser.examples.ContinuousGetMediaWorker;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+
+@Slf4j
+public class Terminate extends ProducerSdkCanaryConsumer implements Runnable {
```

Comment:
Curious  why Terminate?

Reply:
Because I want to run the consumer only for a fixed duration and then exit the application.